application/datamanager/features/steps/equity_bars_steps.py

```
@@ -0,0 +1,167 @@
+@given('I have equity bars data for date "{date_str}"')
+def step_impl(context, date_str):
+    url = f"{context.api_url}/equity-bars"
+    response = requests.post(url, json={"date": date_str})
+    assert response.status_code == 200, f"Failed to set up data for date {date_str}"
+    context.test_date = date_str
+
+
+@when(
+    'I send a GET request to "{endpoint}" with date range "{start_date}" to "{end_date}"'
+)
+def step_impl(context, endpoint, start_date, end_date):
```

Ruff (0.11.9)

Redefinition of unused `step_impl` from line 77

(F811)